src/lib/server/websearch/searchWeb.ts

```diff
@@ -11,7 +12,9 @@ export function getWebSearchProvider() {
 
 // Show result as JSON
 export async function searchWeb(query: string) {
-	if (SERPER_API_KEY) {
+	if (USE_LOCAL_WEBSEARCH) {
+		return await searchWebLocal(query);
+	} else if (SERPER_API_KEY) {
 		return await searchWebSerper(query);
 	}
 	if (YDC_API_KEY) {
```

Comment:
nit: looks like the pattern we use here is `if` not `else if`

Reply:
fixed! thanks for catching